gir/gir.go

```diff
@@ -113,6 +114,34 @@ type PkgRepository struct {
 	Path string // gir file path
 }
 
+// FindGIRFiles finds gir files from the given list of pkgs.
+func FindGIRFiles(pkgs ...string) ([]string, error) {
+	girDirs, err := pkgconfig.GIRDir(pkgs...)
+	if err != nil {
+		return nil, err
+	}
+	visited := make(map[string]struct{}, len(girDirs))
+	var girFiles []string
+	for _, pkg := range pkgs {
```

Comment:
Why is this looped over `pkgs` instead of `girDirs`? So:

```go
for pkg, dir := range girDirs {
  files, err := os.ReadDir(girDir)
  for _, file := range files {
    // ...
  }
}
```

Reply:
So that the order is predictable, which was important to me for some reason.

Reply:
I've added a comment saying as much.